awxkit/awxkit/cli/client.py

```
@@ -156,7 +156,7 @@ def authenticate(self):
                 self.root.connection.login(username, password)
                 self.root.get()
             except Exception as e:
-                raise RuntimeError(f"Basic authentication failed: {str(e)}. " "Verify credentials and network connectivity.") from e
+                raise RuntimeError(f"Basic authentication failed: {str(e)}. Verify credentials and network connectivity.") from e
```

Ruff (0.16.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)